vs/sdl2/build-scripts/androidbuildlibs.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
This should be fixed [upstream](https://github.com/libsdl-org/SDL/tree/main/build-scripts).

Reply:
https://github.com/libsdl-org/SDL/pull/7440